install/letsencrypt.bash

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+userName=$1
+FQDN=$2
+configFile=/etc/letsencrypt/"$FQDN".cli.ini
+
+scriptDir="$(dirname $0)"
+cd $scriptDir
+# Read config file with paths to WP-installs and usernames
+source ../conf
+
+# Create configfolder+file if letsencrypt hasn't run yet
+mkdir -p /etc/letsencrypt
+touch $configFile
+
+# Create a config file for domain
+cat > $configFile << EOF
```

Comment:
The lines between EOF and EOF. EOF sets a marker for when to stop - and when we hit EOF the file ends. This does not need to be EOF but could be "whatever" - but i think that EOF more clearly states what it's for.